src/collective/casestudy/behaviors/address_info.py

```diff
@@ -0,0 +1,46 @@
+from collective.casestudy import _
+from plone.autoform.interfaces import IFormFieldProvider
+from plone.supermodel import model
+from zope import schema
+from zope.interface import provider
+
+
+@provider(IFormFieldProvider)
+class IAddressInfo(model.Schema):
+
+    address = schema.TextLine(
+        title=_("label_address", default="Company Address"), required=False
+    )
+
+    address_2 = schema.TextLine(
+        title=_("label_address_2", default="Additional Info"),
+        description=_("description_address_2", default="i.e: room 2"),
```

Comment:
Avoid Latin abbreviations, and instead use fully spelled out modern language equivalents for better i18n.

If you use "i.e." or "e.g.", then you must add a period after each letter, followed by a comma, making it a continuation remark, and using the proper context, e.g., [as described in "The Oatmeal"](https://theoatmeal.com/comics/ie).

```suggestion
        description=_("description_address_2", default="Example: Room 2"),
```